### PR TITLE
[docs] PR for Add passwordcheck in list of supported extensions

### PR DESCRIPTION
**Pull request by ddhodge**
Added passwordcheck to list of supported PostrgeSQL extensions
#13814

@netlify /preview/explore/ysql-language-features/pg-extensions/

### CODE REVIEW

File: docs/content/preview/explore/ysql-language-features/pg-extensions.md [hunk 1]
```diff
@@ -190,6 +191,40 @@ SELECT levenshtein('Yugabyte', 'yugabyte'), metaphone('yugabyte', 8);
...
+To enable the passwordcheck extension, add `passwordcheck` to `shared_preload_libraries` in the PostgreSQL server configuration parameters using the YB-TServer [`--ysql_pg_conf_csv`](../../../reference/configuration/yb-tserver/#ysql-pg-conf-csv) flag:
+
+```sh
+--ysql_pg_conf_csv="shared_preload_libraries=passwordcheck"
+```
+
+When enabled, if a password is considered too weak, it's rejected with an error. For example:
+
+```sql
+yugabyte=# create role test_role password '1234';
```
Comment (sushantrmishra) [inline docs/content/preview/explore/ysql-language-features/pg-extensions.md:205]: May be use the nicer examples from this file.

https://github.com/yugabyte/yugabyte-db/blob/master/src/postgres/contrib/passwordcheck/sql/passwordcheck.sql

File: docs/content/preview/explore/ysql-language-features/pg-extensions.md [hunk 2]
```diff
@@ -30,14 +30,15 @@ For information about using a specific extension in YugabyteDB, follow the Examp
 | Extension | Status | Description | Examples |
 | :-------- | :----- | :---------- | :------ |
 | [PostgreSQL modules](https://www.postgresql.org/docs/11/contrib.html) |
+| [file_fdw](https://www.postgresql.org/docs/11/file-fdw.html) | Pre-bundled | Provides the foreign-data wrapper file_fdw, which can be used to access data files in the server's file system. | [Example](#file-fdw-example) |
 | [fuzzystrmatch](https://www.postgresql.org/docs/11/fuzzystrmatch.html) | Pre-bundled | Provides several functions to determine similarities and distance between strings. | [Example](#fuzzystrmatch-example) |
+| [hstore](https://www.postgresql.org/docs/11/hstore.html) | Pre-bundled | Implements the hstore data type for storing sets of key/value pairs in a single PostgreSQL value. | |
```
Comment (lizayugabyte) [inline docs/content/preview/explore/ysql-language-features/pg-extensions.md:35]: key-value pairs

File: docs/content/preview/secure/audit-logging/audit-logging-ysql.md [hunk 3]
```diff
@@ -27,35 +27,31 @@ type: docs
...
 ### Step 1. Enable audit logging on YB-TServer
 
 This can be done in one of the following ways:
 
-##### Option A: Using `--ysql_pg_conf` TServer flag
-
-Database administrators can leverage `ysql_pg_conf` to set appropriate values for `pgAudit` configuration.
-
-For example, `ysql_pg_conf="pgaudit.log='DDL',pgaudit.log_level=notice"`
+- Use the `--ysql_pg_conf_csv` TServer flag.
```
Comment (lizayugabyte) [inline docs/content/preview/secure/audit-logging/audit-logging-ysql.md:40]: YB-TServer flag

File: docs/content/preview/secure/audit-logging/audit-logging-ysql.md [hunk 4]
```diff
@@ -65,10 +61,9 @@ Enable audit logging in YugabyteDB clusters by creating the `pgAudit` extension.
 CREATE EXTENSION IF NOT EXISTS pgaudit;
 ```
 
+## Customize audit logging
 
-## Customizing Audit Logging
-
-YSQL audit logging can be further customized by configuring the `pgAudit` flags as per the following table.
+You can further customize YSQL audit logging by configuring the `pgAudit` flags as per the following table.
```
Comment (lizayugabyte) [inline docs/content/preview/secure/audit-logging/audit-logging-ysql.md:66]: flags, as per the following table.

**Review by lizayugabyte — APPROVED** [review]
Minor comments. LGTM

**Review by lizayugabyte — APPROVED** [review]

**Pull request merged**